test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,25 @@
+const axios = require('axios');
+const { expect, assert } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('Consume PUT Method', () => {
+  it('should follow aperdomob', async () => {
+    const response = await axios.put('https://api.github.com/user/following/aperdomob', {}, {
+      headers: {
+        Authorization: `token ${process.env.ACCESS_TOKEN}`
+      }
+    });
+    expect(response.status).to.equal(StatusCodes.NO_CONTENT);
+    expect(response.data).to.equal('');
+  });
+
+  it('verify that I follow aperdomob', async () => {
+    const response = await axios.get('https://api.github.com/user/following', {
+      headers: {
+        Authorization: `token ${process.env.ACCESS_TOKEN}`
+      }
+    });
+    const followingInfo = response.data.find((element) => element.login === 'aperdomob');
+    assert.exists(followingInfo);
+  });
```

Comment:
you need to add one additional test in which you will call again the put method, and then the get method to verify that no matter what amount of times you call the put method to follow an user, it will always produce the same result (aka idempotence)

Reply:
Ok, thanks. I will commit again.